caps/import_utils.py

```diff
@@ -32,7 +93,15 @@ def get_google_sheet_as_csv(key, outfile, sheet_name=None):
         outfile.write(r.content)
 
 
-def replace_csv_headers(csv_file, new_headers, drop_empty_columns=True, outfile=None):
+def replace_csv_headers(
+    csv_file: PathLike,
+    new_headers: list[str],
+    drop_empty_columns: bool = True,
+    outfile: Optional[PathLike] = None,
+):
+    """
+    Downloading a google sheet csv will sometimes mess up the headers - needs manual fix
+    """
```

Comment:
just for clarity could we split the docs/hinting changes into a separate commit?